pkg/kadm/topics.go

```diff
@@ -231,9 +231,10 @@ func (cl *Client) createTopics(ctx context.Context, dry bool, p int32, rf int16,
 
 // DeleteTopicResponse contains the response for an individual deleted topic.
 type DeleteTopicResponse struct {
-	Topic string  // Topic is the topic that was deleted, if not using topic IDs.
-	ID    TopicID // ID is the topic ID for this topic, if talking to Kafka v2.8+ and using topic IDs.
-	Err   error   // Err is any error preventing this topic from being deleted.
+	Topic        string  // Topic is the topic that was deleted, if not using topic IDs.
+	ID           TopicID // ID is the topic ID for this topic, if talking to Kafka v2.8+ and using topic IDs.
+	Err          error   // Err is any error preventing this topic from being deleted.
+	ErrorMessage *string // ErrorMessage a potential extra message describing any error.
```

Comment:
Sure, I can change the field to a string.

fwiw, I used a pointer here since it's the same type used for this field in the kmsg package ([example](https://github.com/twmb/franz-go/blob/master/pkg/kmsg/generated.go#L3169)) and we get the field from there (and it's easier for nil checks when the broker does not return a message).

Reply:
Done, thanks.